### PR TITLE
DNM Migrate elb_classic_lb_info* modules and tests

### PR DESCRIPTION
**Pull request by mandar242**
Migrate elb_classic_lb_info* modules and tests

### CODE REVIEW

File: plugins/modules/elb_classic_lb_info.py [hunk 1]
```diff
@@ -0,0 +1,516 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Contributors to the Ansible project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r"""
+---
+module: elb_classic_lb_info
+version_added: 1.0.0
```
Comment (abikouo) [inline plugins/modules/elb_classic_lb_info.py:10]:
```suggestion
version_added: 1.0.0
version_added_collection: community.aws
```

File: plugins/modules/elb_classic_lb_info.py [hunk 2]
```diff
@@ -0,0 +1,516 @@
...
+# Note: These examples do not set authentication details, see the AWS Guide for details.
+# Output format tries to match amazon.aws.elb_classic_lb module input parameters
+
+- name: Gather information about all ELBs
+  amazon.aws.elb_classic_lb_info:
+  register: elb_info
+
+- ansible.builtin.debug:
+    msg: "{{ item.dns_name }}"
+  loop: "{{ elb_info.elbs }}"
```
Comment (abikouo) [inline plugins/modules/elb_classic_lb_info.py:40]:
```suggestion
```

File: plugins/modules/elb_classic_lb_info.py [hunk 3]
```diff
@@ -0,0 +1,516 @@
...
+
+- name: Gather information about a particular ELB
+  amazon.aws.elb_classic_lb_info:
+    names: frontend-prod-elb
+  register: elb_info
+
+- ansible.builtin.debug:
+    msg: "{{ elb_info.elbs.0.dns_name }}"
```
Comment (abikouo) [inline plugins/modules/elb_classic_lb_info.py:48]:
```suggestion
```

File: plugins/modules/elb_classic_lb_info.py [hunk 4]
```diff
@@ -0,0 +1,516 @@
...
+- name: Gather information about a set of ELBs
+  amazon.aws.elb_classic_lb_info:
+    names:
+      - frontend-prod-elb
+      - backend-prod-elb
+  register: elb_info
+
+- ansible.builtin.debug:
+    msg: "{{ item.dns_name }}"
+  loop: "{{ elb_info.elbs }}"
```
Comment (abikouo) [inline plugins/modules/elb_classic_lb_info.py:59]:
```suggestion
```

File: tests/integration/targets/elb_classic_lb_info/vars/main.yml [hunk 5]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (abikouo) [inline tests/integration/targets/elb_classic_lb_info/vars/main.yml:1]: This file can be deleted

File: tests/integration/targets/elb_classic_lb_info/meta/main.yml [hunk 6]
```diff
@@ -0,0 +1 @@
+dependencies: []
```
Comment (abikouo) [inline tests/integration/targets/elb_classic_lb_info/meta/main.yml:1]: This file can be removed

**Review by abikouo — DISMISSED** [review]

**Review by alinabuzachis — APPROVED** [review]

**Comment by alinabuzachis** [comment]
This must be reverted because we cannot merge the migration PR unless we disable the squash strategy.

**Pull request merged**